### DISCUSSION TITLE
Error while running code in pycharm

### DISCUSSION THREAD

**Discussion by siddharthjena29** [peterdsharpe/AeroSandbox#85] [General]
Hello,
iam a newbie, starting to learn from tutorials. Trying to run the rosenbrock function. Getting this error while running in pycharm.
Kindly help if anyone can.

_C:\Users\sidd\Scripts\python.exe C:\Users\sidd\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch.py 
Traceback (most recent call last):
  File "C:\Users\sidd\AppData\Roaming\JetBrains\PyCharmCE2022.2\scratches\scratch.py", line 1, in <module>
    import asb
ModuleNotFoundError: No module named 'asb'

Process finished with exit code 1_

**Comment by peterdsharpe** [comment]
Hi,

Replace `import asb` with `import aerosandbox as asb`.

**Comment by siddharthjena29** [comment]
Hi Peter,
Thank you for the solution. It works.
This is truly amazing work that you have done with aerosandbox. It's really
helpful for newbies like me. I intend to do lot of research in the field of
wind engineering using this.
Thanks and regards
